### PR TITLE
fix: handle operations labels that are too long

### PR DESCRIPTION
**Pull request by drazik**
Add ellipsis if the operation label overflows : 

image

### CODE REVIEW

**Comment by ptbrowne** [comment]
This feels weird to me :/ Can you check with flo ?

**Comment by drazik** [comment]
He told me that since we already do that on the mobile side, we must do the same on desktop.

File: src/ducks/transactions/Transactions.styl [hunk 1]
```diff
@@ -32,6 +32,9 @@
     .bnk-op-caticon
         cursor pointer
 
+    .bnk-op-label
+        max-width 100%
```
Comment (ptbrowne) [inline src/ducks/transactions/Transactions.styl]: I think this should be done automatically on `media__body`

Comment (ptbrowne) [reply src/ducks/transactions/Transactions.styl]: Can you add a TODO here ?

Comment (ptbrowne) [reply src/ducks/transactions/Transactions.styl]: I don't understand why this is necessary ? Has the ListItemText a problem with Media Bd ?

**Pull request merged**